### PR TITLE
Add new routes on the project

### PR DESCRIPTION
**Pull request by Cyrildab**

### CODE REVIEW

File: frontend/src/components/HomePage/MainHeader/MainHeader.jsx [hunk 1]
```diff
@@ -1,12 +1,15 @@
...
 
 function MainHeader() {
   return (
     <header>
       <h1 className="mainTitle">Le collectionneur temporel</h1>
-      <button type="button" className="aboutButton">
-        A propos
-      </button>
+      <Link to="/aboutus">
+        <button type="button" className="aboutButton">
```
Comment (Yoann-Fortin) [inline frontend/src/components/HomePage/MainHeader/MainHeader.jsx:9]: Why a button in a tag ?

File: frontend/src/components/AboutUs/AboutUsPage.scss [hunk 2]
```diff
@@ -10,3 +12,20 @@ body {
...
+  border: 3px solid $primaryColor;
+  background-color: $secondaryColor;
+  height: 3rem;
+  width: 7vw;
+  align-items: center;
+  color: $primaryColor;
+  font-size: 1rem;
+  margin-right: 4rem;
+  margin-left: 2rem;
+  margin-top: 10px;
```
Comment (Yoann-Fortin) [inline frontend/src/components/AboutUs/AboutUsPage.scss]: Why px also rem or vh or % ?

File: frontend/src/components/AboutUs/AboutUsPage.scss [hunk 3]
```diff
@@ -10,3 +12,20 @@ body {
...
+  cursor: pointer;
+  position: absolute;
+  top: 22px;
```
Comment (Yoann-Fortin) [inline frontend/src/components/AboutUs/AboutUsPage.scss]: Why non responsive unity ?

File: frontend/src/components/ChooseLevelPage/ChooseLevelPage.jsx [hunk 4]
```diff
@@ -0,0 +1,28 @@
+import { Link } from "react-router-dom";
+import "./ChooseLevelPage.scss";
+
+function ChooseLevelPage() {
+  const universe = [
+    { name: "CRÉTACÉ", Id: 1 },
```
Comment (Yoann-Fortin) [inline frontend/src/components/ChooseLevelPage/ChooseLevelPage.jsx]: Id is a class ?

File: frontend/src/components/ChooseLevelPage/ChooseLevelPage.scss [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+  color: $primaryColor;
+  border-inline: 1px solid $primaryColor;
+  background-size: cover;
+  width: 20vw;
+  &1 {
+    background-image: url(../../assets/pictures/Jurassique.png);
```
Comment (Yoann-Fortin) [inline frontend/src/components/ChooseLevelPage/ChooseLevelPage.scss:25]: No Franglais in code

File: frontend/src/components/TargetButton.scss [hunk 6]
```diff
@@ -16,21 +17,26 @@ $cd-txt: orange;
   border-radius: 50px;
   text-align: center;
   box-shadow: 2px 2px 4px rgba(0, 0, 0, 0.2);
-  border: 0.15rem solid green;
+  border: 0.15rem solid #2b160f;
```
Comment (Yoann-Fortin) [inline frontend/src/components/TargetButton.scss]: variable is better

**Review by Bravax — APPROVED** [review]
All clear

File: frontend/src/components/AboutUs/AboutUsPage.jsx [hunk 7]
```diff
@@ -40,17 +41,24 @@ function AboutUsPage() {
...
-          name={e.name}
-          img={e.imgSrc}
-          github={e.github}
-          linkedin={e.linkedin}
-        />
-      ))}
-    </main>
+    <>
+      <Link to="/">
+        <button type="button" className="CloseButtonAboutUs">
```
Comment (Yoann-Fortin) [inline frontend/src/components/AboutUs/AboutUsPage.jsx]: No PascalCase in className

File: frontend/src/components/ChooseLevelPage/ChooseLevelPage.jsx [hunk 8]
```diff
@@ -10,18 +10,25 @@ function ChooseLevelPage() {
...
-              Jouer
-            </button>
-          </Link>
-        </article>
-      ))}
-    </main>
+    <>
+      <main className="cardContainer">
+        {universe.map((world) => (
+          <article key={world.name} className={`card card${world.Id}`}>
```
Comment (Yoann-Fortin) [inline frontend/src/components/ChooseLevelPage/ChooseLevelPage.jsx]: No PascalCase for a props in object

File: frontend/src/components/LevelPage/ReturnGameMenu/ReturnGameMenu.jsx [hunk 9]
```diff
@@ -0,0 +1,44 @@
...
+  }
+  return (
+    <>
+      <button
+        type="button"
+        className="BackMainMenuButton"
+        onKeyDown={HandleEvent}
+        onClick={HandleEvent}
+      >
+        X
```
Comment (Yoann-Fortin) [inline frontend/src/components/LevelPage/ReturnGameMenu/ReturnGameMenu.jsx:38]: Not only, check aria-label

**Review by Yoann-Fortin — APPROVED** [review]

**Pull request merged**